docs/06-concepts/18-testing/01-get-started.md

```diff
@@ -170,24 +180,39 @@ tags:
 
 ```
 
+6. Finally, add the `test` and `serverpod_test` packages as dev dependencies in `pubspec.yaml`:
+
+```yaml
+dev_dependencies:
+  serverpod_test: 2.1.4 # Should be same version as the `serverpod` package
+  test: '>=1.0.0 <2.0.0'
```

Comment:
Does writing it like this make sense?

Reply:
If this is the Dart test library I would write it with a carrot symbol instead (`^`). A range basically only makes sense if we want to allow multiple major versions.